solidity/dashboard/README.md

```diff
@@ -95,7 +95,7 @@ Before MetaMask can be used with `keep-dev` testnet for the first time, it needs
 ### MetaMask KEEP token owner account import
 On `keep-dev`, account `0x0f0977c4161a371b5e5ee6a8f43eb798cd1ae1db` is the owner of contracts including KEEP ERC20 token contract. This account can be used to create token grants and delegate stake to operators. Grantees of tokens can also stake-delegate their grants.
 
-To use this account in the dApp, it needs to be imported from [the JSON file](https://github.com/keep-network/keep-core/blob/master/private-testnet/keyfiles/UTC--2019-03-27T19-05-16.429364100Z--0f0977c4161a371b5e5ee6a8f43eb798cd1ae1db) secured by a [password](https://github.com/keep-network/keep-core/blob/master/private-testnet/eth-account-password.txt).
+To use this account in the dApp, it needs to be imported from [the JSON file](https://github.com/keep-network/keep-core/blob/main/private-testnet/keyfiles/UTC--2019-03-27T19-05-16.429364100Z--0f0977c4161a371b5e5ee6a8f43eb798cd1ae1db) secured by a [password](https://github.com/keep-network/keep-core/blob/main/private-testnet/eth-account-password.txt).
```

Comment:
I just noticed that the link wasn't working before the change and won't be working after the change - there is no `private-testnet` folder in `keep-core`. The whole section of the document looks outdated - I propose to keep the change as it is (i.e. change `master` to `main`) and leave the fixing of this part of document for some later task.